executor/ddl.go

```diff
@@ -66,7 +87,7 @@ func (e *DDLExec) Next(ctx context.Context, chk *chunk.Chunk) (err error) {
 		err = e.executeRenameTable(x)
 	}
 	if err != nil {
```

Comment:
If err == nil, there is no check for schema outdated?

Reply:
Yes, it will be check in ddl_worker.go